react/FilePicker/FilePickerBodyItem.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { memo } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import filesize from 'filesize'
```

Comment:
I don't see `filesize` listed in the package.json 🤔

Reply:
It is a deps of `react-styleguidist>react-dev-utils`
Maybe a future problem ?
Should I install it explicitly in the `cozy-ui` deps?

Reply:
Definitively! 

Since currently it's a side effect to have it, if we remove later `react-styleguidist` or else, we'll break the component. 
The rule: if you directly import a lib, you should install it.